documentation/docs/welcome.md

```diff
@@ -0,0 +1,47 @@
+---
+description: INX-Participation is an extension for nodes to enable on-tangle voting.
+image: /img/Banner/banner_hornet.png
+keywords:
+- IOTA Node
+- Hornet Node
+- INX
+- Participation
+- IOTA
+- Shimmer
+- Node Software
+- Welcome
+- explanation
+---
+
+# Welcome to INX-Participation
+
+INX-Participation is an extension for nodes to enable on-tangle voting. 
```

Comment:
Could you please be more specific? How exactly does it enable the voting? Who is voting?

For example,

"INX-Participation extension handles voting requests from the connected clients and calculates the current voting status for them."